### DISCUSSION TITLE
Is there any way to edit the default text colors for light and dark modes?

### DISCUSSION THREAD

**Discussion by liconver** [quasarframework/quasar#13883] [General - Components / Directives / etc]
I know there isn't a simple configuration for it, and editing the source code there must be a way, but just changing the global scss I haven't found a selector that gives me the desired result.  I tried using the * global selector but caused problems because it overrided whatever color I tried to give to the icons on my buttons.  It seems to be a question of scope.  Does anyone have any experience with this?

**Comment by pratik227** [comment]
https://codepen.io/Pratik__007/pen/zYWrvYr

```
body.body--dark {
  color: ......
}
```

https://quasar.dev/style/dark-mode#how-to-style-your-app

**Reply by liconver** [reply]
Ok thanks, can't remember if I tried that but there still seems to be some kind of an issue with scope I don't understand because that body--  class only seems to effect the top level component and not the child components.  I can assign the color of all the child elements the color I want if they are just regular div's or p tags but if they are q- components changing their color prop doesn't seem to do anything.

**Reply by pratik227** [reply]
Ok But I think this should work for all components

**Reply by liconver** [reply]
Thanks it works for some but for others I have to add the text-black utility class.  But on this q-input, every selector I try the text was still white when dark=true in the config.  What allowed me to change it was setting dark=false in the q-input prop.

**Reply by pratik227** [reply]
https://quasar.dev/style/sass-scss-variables#variables-list

`$input-text-color : rgba(0,0,0,.87) !default`

Yes I think for input that's the option or I think you can change the $input-text-color

**Reply by liconver** [reply]
Thanks a lot for the help I'm just learning quasar now it has a lot of nice features but it can be a bit confusing figuring out how to make it more extensible than what it offers "out of the box"

**Reply by pratik227** [reply]
Great 👍
